CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ endif()
 
 add_library(crossplatform_compiler_flags INTERFACE)
 target_compile_features(crossplatform_compiler_flags INTERFACE cxx_std_11)
-target_compile_options(crossplatform_compiler_flags INTERFACE -std=c++11)
+target_compile_options(crossplatform_compiler_flags INTERFACE -std=c++11 -Werror)
```

Comment:
But this still doesn't catch it, correct? Annoying that only clang does...

Reply:
Hmm, good point. I only tested it with clang. Doing a `gcc` run now

Reply:
Catches it with gnu also